ops/docker/Dockerfile.packages

```diff
@@ -0,0 +1,70 @@
+# This Dockerfile builds all the dependencies needed by the monorepo, and should
+# be used to build any of the follow-on services
+#
+# ### BASE: Install deps
+# We do not use Alpine because there's a regression causing it to be very slow
+# when used with typescript/hardhat: https://github.com/nomiclabs/hardhat/issues/1219
+FROM node:14.18.1-buster-slim as base
+
+RUN apt-get update -y && apt-get install -y git
+
+ARG VERSION=v0.8.9
+ARG SOLC_VERSION=${VERSION}+commit.e5eed63a
+ARG SOLC_UPSTREAM=https://github.com/ethereum/solc-bin/raw/gh-pages/linux-amd64/solc-linux-amd64-${SOLC_VERSION}
+
+ADD $SOLC_UPSTREAM ./solc
+ADD https://github.com/ethereum/solc-bin/raw/gh-pages/linux-amd64/solc-linux-amd64-v0.5.17+commit.d19bba13 ./solc
```

Comment:
I'm unsure if we still need solc `v0.5.17`

Reply:
I'm actually going to remove this altogether. I don't see any demonstrable differences in build time.